src/cheatsheet.component.ts

```diff
@@ -43,17 +43,23 @@ import {Hotkey} from './hotkey.model';
 .cfp-hotkeys {
   width: 100%;
   height: 100%;
-  display: table-cell;
+  display: block;
   vertical-align: middle;
 }
 
+.cfp-hotkeys-table{
```

Comment:
Just for consistency's sake I'd love to see a space before the bracket, like this:
```suggestion
.cfp-hotkeys-table {
```